Merge ping_identity into master (#2229)
DataDog/integrations-extras: ping_identity/README.md

# Ping Identity - Datadog API Connector

## Overview

> Ping Identity delivers intelligent identity solutions for the enterprise. We enable companies to achieve Zero Trust identity-defined security and more personalized, streamlined user experiences.

Reviewer: ```suggestion
Ping Identity delivers intelligent identity solutions for the enterprise. The Ping Identity platform enables companies to achieve Zero Trust identity-defined security and personalized, streamlined user experiences.
```

> Datadog integrates directly with the PingOne DaVinci platform through the Datadog API connector. PingOne DaVinci is an orchestration platform that lets you create flows using connections and logical operators. These flows guide users through defined processes that can present customized pages, modify values, or perform other actions. 

Reviewer 2: A few initial comments:

Do you have a more concrete example of what a user could build with DaVinci?
Can you also add more color into the kinds of actions users can perform with Datadog in DaVinci?

> The Datadog API connector boasts over 350+ unique capabilites, each with a unique Datadag API endpoint to complete CRUD actions for many different Datadog components.

Reviewer: ```suggestion
The Datadog API connector has over 350+ unique capabilities, each with a unique Datadog API endpoint to complete CRUD actions for many different Datadog components.
```

## Setup

### Installation

> Installation with PingOne's DaVinci platform is simple and easy with Ping's drag and drop functionality on a flow canvas. Follow the below steps to configure Datadog within DaVinci:

Reviewer: ```suggestion
Installation with PingOne's DaVinci platform uses Ping's drag and drop functionality on a flow canvas. Follow the below steps to configure Datadog within DaVinci:
```

> 1. On a flow, select the black "+" button in the bottom left corner of the flow canvas. 

Reviewer: ```suggestion
1. On a flow, select the black **+** button in the bottom left corner of the flow canvas. 
```

> 2. Search for Datadog and select it to add it to the canvas.

Reviewer: ```suggestion
2. Search for "Datadog" and select it to add it to the canvas.
```
Can you replace "it" with a word that describes how "Datadog" appears? Is it a list item, a button, a tile, or something else?

### Configuration

> 1. Click into the connector to open the connector side panel. Select "Configure". You will need to enter the API URL, Authentication API Key and Authentication Application Key from your Datadog tenant to successfully configure the connector within your DaVinci tenant. "Apply" changes once added.
> 
> 2. Scroll and select the proper capability for your use case.
> 
> 3. Enter any additional information required for the connector to successfuly run. These will be capability dependent.

Reviewer: ```suggestion
1. Click the connector to open the connector side panel. 
1. Select **Configure**.
1. Enter the API URL, API key, and application key from your Datadog tenant to configure the connector within your DaVinci tenant.
1. Click **Apply**.
1. Scroll and select the proper capability for your use case.
1. Enter any additional information required for the connector to successfully run. These are capability dependent.
```

### Validation

> You can validate that the connector is working by linking the connector to other connectors (namely the HTTP connector) to view the output of the connector in a demo flow. Utilize the "Custom HTML Message" capability from the HTTP connector and in the message field, select the "{}" to choose the output from the Datadog connector. Run the flow widget by selecting "Try Flow" in the upper right corner to execute the connector and determine if it is working as expected.

Reviewer: ```suggestion
You can validate that the connector is working by linking the connector to the HTTP connector to view the output of the connector in a demo flow:
1. Use the **Custom HTML Message** capability from the HTTP connector.
1. In the message field, select **{}** to choose the output from the Datadog connector.
1. Run the flow widget by selecting **Try Flow** in the upper right corner. The connector executes.
1.  <tell the user what to look for to determine if the connector is working correctly>
```

## Data Collected

### Metrics

Ping Identity does not include any metrics.

### Service Checks

Ping Identity does not include any service checks.

### Events

Ping Identity does not include any events.

## Troubleshooting

Need help? Contact tap-global@pingidentity.com.

> You can view general information on the DaVinci platform [here](https://docs.pingidentity.com/r/en-us/davinci/davinci_landing_page).

Reviewer: ```suggestion
See [PingOne DaVinci](https://docs.pingidentity.com/r/en-us/davinci/davinci_landing_page) for general information on the DaVinci platform.
```
